blue_heron/lib/blue_heron/address.ex

```diff
@@ -0,0 +1,83 @@
+defmodule BlueHeron.Address do
+  @moduledoc """
+  Helper struct for working with BLE Addresses.
+  """
+
+  alias BlueHeron.Address
+
+  defstruct string: "00:00:00:00:00:00",
+            binary: <<00, 00, 00, 00, 00, 00>>,
+            integer: 0x000000000000
+
+  defimpl Inspect do
+    def inspect(address, _opts) do
+      "#Address<#{address.string}>"
```

Comment:
```suggestion
       "#BlueHeron.Address<#{address.string}>"
```

I think including the `BlueHeron` prefix to the struct name would make this more understandable when inspected in a larger application. Also this follows the pattern of `Ecto.Query`, `Ecto.Changeset` and others.

Reply:
I like this and would totally approve any PR that comes for it 😉

Reply:
Okay, I'll try and make one later today